### PR TITLE
New Footer & Secondary / Settings Navigation

### PR DESCRIPTION
**Pull request by biyeun**
@sravfeyn @orangejenny 
cc @dimagi/product 
screen shot 2016-02-01 at 6 58 02 pm
screen shot 2016-02-01 at 6 57 55 pm
screen shot 2016-02-01 at 6 57 49 pm

### CODE REVIEW

**Comment by orangejenny** [comment]
sleek!

(haven't reviewed, just observing)

**Comment by proteusvacuum** [comment]
Sooo hot! 
One comment that I've seen in the wild: when people put their name as "Dr. Firstname Lastname", their `firstname` gets set to Dr, which then shows Dr as their name for the dropdown, which is a little weird. Would it be terrible to add the full name to the dropdown (and maybe truncate with ellipses if too long)?

**Comment by czue** [comment]
:heart_eyes:

**Comment by millerdev** [comment]
:heart:

**Comment by biyeun** [comment]
@proteusvacuum 
screen shot 2016-01-04 at 4 38 32 pm

**Comment by biyeun** [comment]
on staging

**Comment by proteusvacuum** [comment]
<3

**Comment by amsagoff** [comment]
hallelujah!

My only feedback is that in the settings dropdown there are a lot of
different fonts/font sizing going on, which looks a bit visually confusing
compared to the cleanness of the other dropdowns.

**Comment by amsagoff** [comment]
and last small thing: I propose changing "Current Subscription" to "Manage
Subscription" (can still land you on the Current Subscription page).

File: corehq/apps/hqwebapp/templatetags/hq_shared_tags.py [hunk 1]
```diff
@@ -146,13 +146,15 @@ def domains_for_user(context, request, selected_domain=None):
         'is_public': selected_domain == 'public',
         'domain_list': domain_list,
         'current_domain': selected_domain,
-        'DOMAIN_TYPE': context['DOMAIN_TYPE']
+        'DOMAIN_TYPE': context['DOMAIN_TYPE'],
+        'can_publish_to_exchange': (
+            selected_domain is not None and selected_domain != 'public'
+            and request.couch_user and request.couch_user.can_edit_apps() and
+                (request.couch_user.is_member_of(selected_domain)
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/templatetags/hq_shared_tags.py]: E131 continuation line unaligned for hanging indent

**Comment by biyeun** [comment]
@dimagi/product what's the UAT status on this...it's updated on staging. Are we QAing?

**Comment by amsagoff** [comment]
Good to go by me - would be good to get QA to sign off (I think it's been
through?)

**Comment by biyeun** [comment]
QA Passed! :white_check_mark:

**Comment by orangejenny** [comment]
HQ getting more beautiful all the time.

**Comment by amsagoff** [comment]
LIBERATE THE HORIZONTAL SPACE!

**Comment by czue** [comment]
This is fantastic! :city_sunrise:

**Pull request merged**